include/firestarter/Environment/X86/Platform/SapphireRapidsConfig.hpp

```diff
@@ -0,0 +1,52 @@
+/******************************************************************************
+ * FIRESTARTER - A Processor Stress Test Utility
+ * Copyright (C) 2020 TU Dresden, Center for Information Services and High
+ * Performance Computing
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/\>.
+ *
+ * Contact: daniel.hackenberg@tu-dresden.de
+ *****************************************************************************/
+
+#pragma once
+
+#include <firestarter/Environment/X86/Payload/AVX512_AMX_Payload.hpp>
+#include <firestarter/Environment/X86/Platform/X86PlatformConfig.hpp>
+
+namespace firestarter::environment::x86::platform {
+class SapphireRapidsConfig final : public X86PlatformConfig {
+
+public:
+  SapphireRapidsConfig(asmjit::CpuFeatures const &supportedFeatures,
+                  unsigned family, unsigned model, unsigned threads)
+      : X86PlatformConfig("SKL_XEONEP", 6, {85}, {1, 2}, 0,
```

Comment:
The first string should represent a short hand for Sapphire Rapids, this one is for Skylake.
Family and Model should be adjusted to 6 and 143 respectively. Search for CPUID on https://en.wikichip.org/wiki/intel/microarchitectures/sapphire_rapids

Reply:
I updated the string, family and model.